spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -4,21 +4,33 @@
 
 RSpec.describe Reek::Spec::ShouldReekOf do
   describe 'smell type check' do
-    let(:src) { 'def double_thing() @other.thing.foo + @other.thing.foo end' }
+    let(:ruby) { 'def double_thing() @other.thing.foo + @other.thing.foo end' }
 
     it 'reports duplicate calls' do
-      expect(src).to reek_of(:Duplication)
+      expect(ruby).to reek_of(:Duplication)
     end
 
     it 'does not report any other smell types' do
-      expect(src).not_to reek_of(:FeatureEnvy)
+      expect(ruby).not_to reek_of(:FeatureEnvy)
+    end
+
+    it 'reports duplicate calls to @other.thing' do
```

Comment:
Why are the next two examples introduced here? It seems like they are kind of copied from https://github.com/troessner/reek/blob/master/spec/reek/smells/duplicate_method_call_spec.rb ?
You can safely remove those.